diff --git a/node_modules/lighthouse/types/artifacts.d.ts b/node_modules/lighthouse/types/artifacts.d.ts
index b5862be..0370efa 100644
--- a/node_modules/lighthouse/types/artifacts.d.ts
+++ b/node_modules/lighthouse/types/artifacts.d.ts
@@ -10,8 +10,8 @@ import {parseManifest} from '../core/lib/manifest-parser.js';
 import {Simulator} from '../core/lib/dependency-graph/simulator/simulator.js';
 import {LighthouseError} from '../core/lib/lh-error.js';
 import {NetworkRequest as _NetworkRequest} from '../core/lib/network-request.js';
-import speedline from 'speedline-core';
-import TextSourceMap from '../core/lib/cdt/generated/SourceMap.js';
+import * as speedline from 'speedline-core';
+import * as TextSourceMap from '../core/lib/cdt/generated/SourceMap.js';
 import {ArbitraryEqualityMap} from '../core/lib/arbitrary-equality-map.js';
 import type { TaskNode as _TaskNode } from '../core/lib/tracehouse/main-thread-tasks.js';
 import AuditDetails from './lhr/audit-details.js'
