diff --git a/node_modules/lighthouse/core/lib/lh-error.d.ts b/node_modules/lighthouse/core/lib/lh-error.d.ts
index dbe4323..8db2d62 100644
--- a/node_modules/lighthouse/core/lib/lh-error.d.ts
+++ b/node_modules/lighthouse/core/lib/lh-error.d.ts
@@ -61,7 +61,7 @@ export class LighthouseError extends Error {
      * @param {Record<string, string|undefined>=} properties
      * @param {ErrorOptions=} options
      */
-    constructor(errorDefinition: LighthouseErrorDefinition, properties?: Record<string, string | undefined> | undefined, options?: ErrorOptions | undefined);
+    constructor(errorDefinition: LighthouseErrorDefinition, properties?: Record<string, string | undefined> | undefined, options?: any | undefined);
     code: string;
     friendlyMessage: import("../index.js").IcuMessage;
     lhrRuntimeError: boolean;
diff --git a/node_modules/lighthouse/types/artifacts.d.ts b/node_modules/lighthouse/types/artifacts.d.ts
index 7560568..207eeaa 100644
--- a/node_modules/lighthouse/types/artifacts.d.ts
+++ b/node_modules/lighthouse/types/artifacts.d.ts
@@ -10,7 +10,7 @@ import {parseManifest} from '../core/lib/manifest-parser.js';
 import {Simulator} from '../core/lib/dependency-graph/simulator/simulator.js';
 import {LighthouseError} from '../core/lib/lh-error.js';
 import {NetworkRequest as _NetworkRequest} from '../core/lib/network-request.js';
-import speedline from 'speedline-core';
+import * as speedline from 'speedline-core';
 import * as TextSourceMap from '../core/lib/cdt/generated/SourceMap.js';
 import {ArbitraryEqualityMap} from '../core/lib/arbitrary-equality-map.js';
 import type { TaskNode as _TaskNode } from '../core/lib/tracehouse/main-thread-tasks.js';
